genbindings.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Generate (and reasonably test) C bindings
+
+# First wipe all the existing C bindings:
+rm lightning-c-bindings/src/{ln,util,chain,routing}/*
```

Comment:
Is it possible for `lightning-c-bindings/src/{ln,util,chain,routing}` to contain dotfiles/directories? Perhaps:
```suggestion
rm -rf lightning-c-bindings/src/{ln,util,chain,routing}
mkdir -p lightning-c-bindings/src/{ln,util,chain,routing}
```